docs/migration-to-NRI.md

```diff
@@ -0,0 +1,182 @@
+# Prerequisities
+- Up and running CRI Resource Manager
+- One of the two supported policies in use: balloons or topology-aware.
+- For other policies a little bit more work is required and the policies need to be 'ported'. This can be done by just following the example of how balloons or topology-aware was converted.
+
+# Steps for an initial/basic migration test
+## Containerd 
+- Replace the containerd version in the system with 1.7 or newer version (NRI server not supported in older versions).
+
+- Get the Kubelet args:
+  ```console
+  systemctl cat kubelet <- Look for: EnvironmentFile=/.../kubeadm-flags.env
+  ```
+
+- Change the Kubelet container runtime endpoint to the containerd socket:
+    ```console 
+    $ sudo vim /.../kubeadm-flags.env
+    KUBELET_KUBEADM_ARGS="--container-runtime-endpoint=unix:///var/run/containerd/containerd.sock --pod-infra-container-image=registry.k8s.io/pause:3.9"
+
+    $ sudo vim /etc/sysconfig/kubelet 
+    KUBELET_EXTRA_ARGS= --container-runtime-endpoint=/var/run/containerd/containerd.sock <- Remember this aswell
+
+    $ systemctl restart containerd
+    $ systemctl restart kubelet
+    ```
+
+- Edit the containerd config file and look for the section [plugins."io.containerd.nri.v1.nri"] and replace "disable = true" with "disable = false":
+  ```console
+  $ vim /etc/containerd/config.toml
+  ```
+  ```toml
+  [plugins."io.containerd.nri.v1.nri"]
+    disable = false
+    disable_connections = false
+    plugin_config_path = "/etc/nri/conf.d"
+    plugin_path = "/opt/nri/plugins"
+    plugin_registration_timeout = "5s"
+    plugin_request_timeout = "2s"
+    socket_path = "/var/run/nri/nri.sock"
+  ```
+
+## CRI-O
+- Replace the crio version in the system with 1.26.2 or newer.
+
+- Same thing with the Kubelet runtime endpoint, but change it to the crio socket and then restart the crio service.
```

Comment:
What same thing? I think it's best just to spell out the detailed steps here

Reply:
+1